tools/install.sh

```diff
@@ -91,7 +91,7 @@ if [ -z "$LINUX_VERSION" ]; then
 	echo "  - BackBox"
 	echo "  - CentOS"
 	echo "  - Debian"
-	echo "  - Fedora"
+	echo "  - Fedora Release 22-23"
```

Comment:
I've purposely avoided listing specific versions so that less code needs to be updated when the versions change. This should be consistent with the others and just say Fedora. The user should understand that the version is the latest and we should ensure that is accurate.